src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx

```diff
@@ -11,10 +11,10 @@ import ErrorBlackIcon from "../../../../images/error_black.svg";
 import infoBubbleIcon from "../../../../images/info-bubble-9x9.svg";
 import InfoIcon from "../../../../images/info_color.svg";
 import ErrorRedIcon from "../../../../images/error_red.svg";
-import LoaderIcon from "../../../../images/circular_loader_black_16x16.gif";
 import ClearIcon from "../../../../images/Clear.svg";
-import ChevronUpIcon from "../../../../images/QueryBuilder/CollapseChevronUp_16x.png";
-import ChevronDownIcon from "../../../../images/QueryBuilder/CollapseChevronDown_16x.png";
+const LoaderIcon = require("../../../../images/circular_loader_black_16x16.gif");
```

Comment:
These are not correct. We shoud be able to use `import`

Reply:
change it back, It can works now with import